### PR TITLE
Add .pdf taraget generation

### PR DESCRIPTION
**Pull request by gerth2**
Closes https://github.com/PhotonVision/photonvision/issues/268 by adding a dynamically-generated .pdf target, built from the current calibration settings.

### CODE REVIEW

**Comment by gerth2** [comment]
Per https://discord.com/channels/725836368059826228/725846797339066379/906716152221294652 - not sure what's up with dot grid. The transform from what the user asks for, versus what grid should actually be produced is... non-trivial.

File: photon-client/src/views/CamerasView.vue [hunk 1]
```diff
@@ -486,9 +493,96 @@ export default {
...
+              segIdx++;
+            }
+
+
+            ///////////////////////////////////////////
+            // Annotate what was drawn + branding
+            doc.setFont('MuseoModerno');
+            doc.text("PhotonVision", 1.0, 1.0, {maxWidth:(paper_x - 2.0)/2, align:"left"});
+            doc.setFont('Prompt-Regular');
+            doc.text(annotation, paper_x-1.0, 1.0, {maxWidth:(paper_x - 2.0)/2, align:"right"});
```
Comment (mdurrani808) [inline photon-client/src/views/CamerasView.vue:582]: Per https://github.com/PhotonVision/photonvision-branding/blob/master/PhotonVision%20Branding/Documentation/Photon%20Vision%20Branding%20Standards.pdf

Museo Moderno is restricted to logo usage only with Prompt for heading / title and Lato for body. This should be changed accordingly if possible.

B/W version of the logo w/ Museo Moderno are avaliable here:  https://github.com/PhotonVision/photonvision-branding/blob/master/PhotonVision%20Branding/Logos/Header/PNG/PhotonVision-Header-singleColor.png

**Comment by gerth2** [comment]
Sample Output:
image

File: photon-server/src/main/resources/web/index.html [hunk 2]
```diff
@@ -1 +1 @@
-<p>UI has not been copied!</p>
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/favicon.png><title>PhotonVision</title><link href=/js/chunk-2d216214.4302abb0.js rel=prefetch><link href=/js/chunk-2d216257.0de4b8cc.js rel=prefetch><link href=/js/chunk-3fcde5ae.5b5de13a.js rel=prefetch><link href=/css/app.7bdd180f.css rel=preload as=style><link href=/css/chunk-vendors.25378257.css rel=preload as=style><link href=/js/app.39ceb90d.js rel=preload as=script><link href=/js/chunk-vendors.8b159492.js rel=preload as=script><link href=/css/chunk-vendors.25378257.css rel=stylesheet><link href=/css/app.7bdd180f.css rel=stylesheet></head><body><noscript><strong>We're sorry but PhotonVision doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.8b159492.js></script><script src=/js/app.39ceb90d.js></script></body></html>
```
Comment (mcm001) [inline photon-server/src/main/resources/web/index.html:1]: This shouldn't be committed

File: photon-client/src/views/CamerasView.vue (hunk 1 above)
Comment (gerth2) [reply photon-client/src/views/CamerasView.vue:582]: changed to real logo!

File: photon-server/src/main/resources/web/index.html (hunk 2 above)
Comment (gerth2) [reply photon-server/src/main/resources/web/index.html]: reverted

**Review by mcm001 — APPROVED** [review]

**Pull request merged**